### ISSUE TITLE
My loss can not reach 0.65

### ISSUE THREAD

**Issue by huawenqing** [Issam28/Brain-tumor-segmentation#20]
The smallest loss I got is 0.735. Is there something wrong with my data(Brats2018) or paramters, such as batchsize.

**Comment by nagwa1** [comment]
how many epoch used and  how many patches if you increase them it will improve the result

**Comment by muween** [comment]
> how many epoch used and how many patches if you increase them it will improve the result

But it didn't work

**Comment by mahaanis** [comment]
Did you use pre-trained model or trained the model from scratch?
